src/config/mod.rs

```diff
@@ -2,29 +2,33 @@ mod audio;
 mod effect;
 mod midi;
 
+pub use audio::Audio;
 pub use effect::{DelayConfig, Effect};
 pub use midi::{Midi, MidiSlider};
 
 use crate::Result;
-use audio::Audio;
 use serde::Deserialize;
 
 #[derive(Debug, Deserialize)]
 pub struct Config {
-    pub audio: Option<Audio>,
-    pub midi: Option<Midi>,
+    #[serde(default)]
+    pub audio: Audio,
```

Comment:
The `serde(default)` tag here will use the `Default` impl of `Audio` when it's not present in the config file, which I think makes more sense than having it be an `Option`.